docs/csharp/language-reference/operators/member-access-operators.md

```diff
@@ -132,6 +132,9 @@ In the preceding example, if you don't use the `??` operator, `numbers?.Length <
 
 The null-conditional member access operator `?.` is also known as the Elvis operator.
 
+> [!NOTE]
+> In C# 8, the null-conditional operators interacts with the [null-forgiving operator](null-forgiving.md) in an unexpected way. For example, the expression `x?.y!.z` is parsed as `(x?.y)!.z`. Due to this interpretation, `z` is evaluated even if `x` is `null`, which may result in a <xref:System.NullReferenceException>.
```

Comment:
@BillWagner Sorry, I did not look at the last commit yesterday and a grammar error crept in: "operators interacts".  Also I rather agree with @pkulikov's comment:
> After all those discussions, I wouldn't call that behavior "unexpected". (That is, the docs shouldn't use subjective language.)

I would prefer a neutral description like this:
> In C# 8, the [null-forgiving operator](null-forgiving.md) terminates the list of preceding null-conditional operations. For example, the expression `x?.y!.z` is parsed as `(x?.y)!.z`. Due to this interpretation, `z` is evaluated even if `x` is `null`, which may result in a <xref:System.NullReferenceException>.

I changed only the first sentence to describe "unexpected interaction" in more precise words. If you consider that an improvement, please let me know if I should create a PR for that.

Reply:
Thanks @AntonLapounov I like that improved wording. I'll watch for a PR.